install/migrations/update_10.0.x_to_11.0.0/assets.php

```diff
@@ -67,6 +68,7 @@
 } else {
     foreach (['profiles', 'translations'] as $field) {
         $migration->addField('glpi_assets_assetdefinitions', $field, 'JSON NOT NULL', ['update' => "'[]'"]);
+        $migration->addField('glpi_assets_assetdefinitions', 'picture', 'text');
```

Comment:
This migration does not work:
![image](https://github.com/glpi-project/glpi/assets/42734840/cf2c3792-ba34-43e4-8730-3ff6f8f5c11f)

It seems the query is done in an unrelated `foreach` loop and is thus being executed twice.